lib/CustomFields/pages/EditCustomFieldsSettings/EditCustomFieldsSettings.js

```diff
@@ -36,18 +42,23 @@ const EditCustomFieldsSettings = ({
   backendModuleId,
   entityType,
   redirectToView,
+  backendModuleName,
+  intl,
 }) => {
   const {
     customFields,
     customFieldsLoaded,
-    hasError,
+    customFieldsFetchFailed,
   } = useCustomFieldsFetch(okapi, backendModuleId, entityType);
-
+  const {
+    sectionTitle,
+    sectionTitleFetchFailed,
+  } = useSectionTitleFetch(okapi, backendModuleName.toUpperCase());
```

Comment:
We also need to check if section title is loaded here: https://github.com/folio-org/stripes-smart-components/pull/725/files#diff-47bbc820a330f2283c6d6ead7954b1e0R127
Currently, you display the default section title until the response is 
The same applies to `ViewCustomFieldsSettings` component